business/noticeService.js

```diff
@@ -30,7 +30,7 @@ class NoticeService {
       content,
       summary: {
         total: coordinates.length,
-        missing: packages.noDefinition.length,
```

Comment:
prehaps we should just remove `missing` altogether, `warnings.noDefinition.length` is equivalent to the original and your new version could also subtract `warnings.noLicense.length`